packages/create-bison-app/template/tests/e2e/playwrightExamples/example.play.ts

```diff
@@ -0,0 +1,20 @@
+import { test, expect } from '@playwright/test';
```

Comment:
These examples might be ok to delete unless we think they are helpful beyond the other tests we have.

Reply:
I was torn -- the tests we have barely do anything. Simply test that the `storageSession` worked. 
If it's okay, I may leave for now until I get some broader E2E tests with NextAuth SSO in.